# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpfe/appshell/AppWindow.cpp
# Commit: 85194e5b7a9e
# Full Hash: 85194e5b7a9e6ed9114ee46aef0b5efb819cca9b
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-04 03:56:33
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part9: Update DOMLocalization to match Localization constructor. r=nika,emilio
#   
#   Depends on D113691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113692
# ==============================================================================

diff -r 181487bdf6c0 -r 85194e5b7a9e xpfe/appshell/AppWindow.cpp
--- a/xpfe/appshell/AppWindow.cpp	Tue Aug 03 16:25:13 2021 +0000
+++ b/xpfe/appshell/AppWindow.cpp	Tue Aug 03 16:25:13 2021 +0000
@@ -113,9 +113,11 @@
 using dom::BrowserHost;
 using dom::BrowsingContext;
 using dom::Document;
+using dom::DocumentL10n;
 using dom::Element;
 using dom::EventTarget;
 using dom::LoadURIOptions;
+using dom::Promise;
 
 AppWindow::AppWindow(uint32_t aChromeFlags)
     : mChromeTreeOwner(nullptr),
